### PR TITLE
Inform the user about the p12 password

### PR DESCRIPTION
**Pull request by yassineim**
Also make it less silly

### CODE REVIEW

File: cert.go [hunk 1]
```diff
@@ -95,7 +95,7 @@ func (m *mkcert) makeCert(hosts []string) {
 		fatalIfErr(err, "failed to save certificate key")
 	} else {
 		domainCert, _ := x509.ParseCertificate(cert)
-		pfxData, err := pkcs12.Encode(rand.Reader, priv, domainCert, []*x509.Certificate{m.caCert}, "changeit")
+		pfxData, err := pkcs12.Encode(rand.Reader, priv, domainCert, []*x509.Certificate{m.caCert}, "123456")
```
Comment (adamdecaf) [inline cert.go]: I don't think we can change the password as there are already lots of installs using `changeit`.

**Comment by yassineim** [comment]
Sorry for the mess, I think it's all good now.

**Comment by adamdecaf** [comment]
Thanks 👍  Could you squash the commits?

**Comment by yassineim** [comment]
Sorry I don't know how to do that.

**Comment by FiloSottile** [comment]
Thanks for the PR, I made a similar change in 6060e20.

**Pull request closed without merging**